WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListViewModel.kt

```diff
@@ -392,6 +420,23 @@ class PostListViewModel @Inject constructor(
         _snackbarAction.postValue(snackbarHolder)
     }
 
+    private fun onRemoteCopyLoaded(post: PostModel) {
+        val undoAction = {
+            // here replace the post with whatever we had before, again
+            if (originalPostCopyForConflictUndo != null) {
+                dispatcher.dispatch(PostActionBuilder.newUpdatePostAction(originalPostCopyForConflictUndo))
+            }
+        }
+        val onDismissAction = {
+            // no op
```

Comment:
I don't think setting variables to null are actually considered `// no op`. I think it literally means this doesn't do anything which is not the case here.

Reply:
removed in 7a47a545b2